docs/add_release_vintage.md

```diff
@@ -0,0 +1,203 @@
+
+Follow the below instructions to prepare a vintage cluster release template CRs in the repository. These CRs provide a
+[Cluster Release Template](./add_release.md) that is later used to create Cluster Definitions.
+
+The instructions below take into account that CAPI like objects are frequently versioned (so you have `v1beta1`, `v1beta2`),
+but often the changes between the versions are rather small and in practice sometimes don't even affect you.
+
+Since CAPI objects are big in general, in this repo we have extracted a common base for some of them. This base is to
+share what is common in CAPI objects across many versions. To make such bases useful, we're introducing another base on
+top of them to set specific CAPI versions and version specific properties on top of the shared base.
+
+*Note:* As always, instructions here respect the
+[repository structure](https://github.com/giantswarm/gitops-template/blob/main/docs/repo_structure.md).
+
+## Example
+
+An example of a WC cluster template created using the vintage API is available in [bases/clusters/aws/v1alpha3](/bases/clusters/aws/v1alpha3/).
+
+## About bases
+
+In order to avoid code duplication, it is advised to utilize the
+[bases and overlays concept](https://kubernetes.io/docs/tasks/manage-kubernetes-objects/kustomization/#bases-and-overlays)
+of Kustomize in order to configure clusters and node pools.
+
+## Create shared template base (optional)
+
+**IMPORTANT:** template base cannot serve as a standalone base for cluster creation, it is there to abstract
+configuration common across the API versions, and is then used as a base to other bases, which provide an overlay with a
+specific configuration. This is to avoid code duplication across bases. If a template base for your provider is already
+here, you may skip this part.
+
+**Bear in mind**, this is not a complete guide of how to create a perfect base, but rather a mere summary of basic steps
+needed to move forward. Hence, instructions here will not always be precise in telling you what to change, as this can
+strongly depend on resources involved, how much out of them you would like to include into a base, etc.
+
+1. Export provider' name and API version of cluster and node pools resources you are about to create, for example `aws`
+and `v1alpha3`. Template base is not tied to any specific version, though we need to specify it for the `kubectl gs`:
+
+    ```sh
+    export PROVIDER=aws
+    export API_VERSION=v1alpha3
+    ```
+
+1. Create a directory structure:
+
+    ```sh
+    mkdir -p release_templates/vintage/${PROVIDER}/template
+    mkdir -p release_templates/vintage/${PROVIDER}/template
+    ```
+
+1. Use the [kubectl gs template cluster](https://docs.giantswarm.io/ui-api/kubectl-gs/template-cluster/) to template
+cluster resources, see example for the `aws` provider below. Use arbitrary values for the mandatory fields:
+
+    ```sh
+    kubectl gs template cluster --provider ${PROVIDER} \
```

Comment:
Who is the target audience for this docs? I feel like this is dangerous advice. As discussed with @ljakimczuk before: `kubectl gs template` defaults a _lot_ of values to empty strings which cause problems later on unless removed. If a customer is to follow this, then we instantly have to take manual action to remove those empty values from the gitops base.

Additionally: Does this command actually work with an non-existent org? I thought we checked this, but could be wrong.